specifications/api/swagger.yaml

```diff
@@ -798,16 +798,21 @@ paths:
       tags: ["Wallets"]
       summary: List
       description: |
-        Priority [Very High]
+        <p align="right">status: <strong>incomplete¹</strong></p>
 
         Return a list of known wallets, ordered from oldest to newest.
+
+        > (1) Wallets are currently returned without any particular ordering.
```

Comment:
Hi @KtorZ. A quick question: don't the following lines contradict one another?

>  Return a list of known wallets, **ordered from oldest to newest.**
> Wallets are currently returned **without any particular ordering.**

Reply:
Yes. Hence the "incomplete" status :sweat_smile:

Reply:
@KtorZ Fair enough.
One more question: is it actually desirable that we (eventually) return wallets in order from oldest to newest? Where does this requirement come from? I see there is a related issue here: https://github.com/input-output-hk/cardano-wallet/issues/250

Reply:
The requirement comes from Daedalus, as the first consumers of this API they requested this and, I believe it makes sense from the API-perspective to have some sort of ordering available when listing resources on a particular endpoint such that clients may expect the same response from the same request and somewhat predict what the answer should be.

Reply:
Thanks for clarifying!